File: saas-admin/src/types/feature.ts
import { Status } from "./app";

export type FeatureDto = {
  id: string;
  status: Status;
  createdDate: string;
  modifiedDate: string;
  autoID: number;
  appId: string;
  key: string;
  name: string;
  unit: string;
  description: string;
};

File: SaasV2.Entity/Feature.cs
// Feature.cs
using System;

namespace SaasV2.Entity
{
    // Uygulamalarda ortak/özel kullanılabilecek özellik tanımı
    public class Feature : BaseEntity
    {
        public Guid AppId { get; set; }          // App-scoped
        public string Key { get; set; }          // Örn: "api.requests", "storage.gb"
        public string Name { get; set; }         // İnsan-dostu ad
        public string Unit { get; set; }         // Örn: "request", "GB", "seat"
        public string Description { get; set; }
    }
}
